src/main/java/com/keeper/homepage/domain/election/application/convenience/ValidCandidateService.java

```diff
@@ -0,0 +1,40 @@
+package com.keeper.homepage.domain.election.application.convenience;
+
+import static com.keeper.homepage.global.error.ErrorCode.ELECTION_CANDIDATE_CANNOT_REGISTER;
+import static com.keeper.homepage.global.error.ErrorCode.ELECTION_CANDIDATE_NOT_FOUND;
+import static com.keeper.homepage.global.error.ErrorCode.MEMBER_JOB_NOT_FOUND;
+
+import com.keeper.homepage.domain.election.dao.ElectionCandidateRepository;
+import com.keeper.homepage.domain.election.entity.ElectionCandidate;
+import com.keeper.homepage.domain.member.dao.role.MemberJobRepository;
+import com.keeper.homepage.domain.member.entity.job.MemberJob;
+import com.keeper.homepage.global.error.BusinessException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class ValidCandidateService {
+
+  private static final long VIRTUAL_ELECTION_CANDIDATE_ID = 1;
+
+  private final ElectionCandidateRepository electionCandidateRepository;
+
+  private final MemberJobRepository memberJobRepository;
```

Comment:
별 이유가 없다면 개행은 의미있게 사용하는게 좋습니다.

몇몇 부분에서 개행문제가 보이던데 다른 코드에도 적용 부탁드려요~

```suggestion
  private static final long VIRTUAL_ELECTION_CANDIDATE_ID = 1;
  
  private final ElectionCandidateRepository electionCandidateRepository;
  private final MemberJobRepository memberJobRepository;
```

Reply:
어려운 개행...

Reply:
개행.. 역시 어렵습니다..ㅎ